ports/serioussam-tfe/sstfe/serioussamtriggers.gptk

```diff
@@ -0,0 +1,40 @@
+back = esc
+start = enter
+
+a = .
+b = ,
+x = e
+y = c
+
+l2 = space
+l1 = esc
+
+r1 = leftalt
+r2 = mouse_left
+
+up = up
+down = down
+left = left
+right = right
+
+left_analog_up = w
+left_analog_down = s
+left_analog_left = a
+left_analog_right = d
+
+l3 = tab
+
+right_analog_up = mouse_movement_up
+right_analog_down = mouse_movement_down
+right_analog_left = mouse_movement_left
+right_analog_right = mouse_movement_right
+
+r3 = home
+
+//deadzone_triggers = 0
+mouse_scale = 6144
+
+mouse_delay = 16
+
+//deadzone_y = 2100
+//deadzone_x = 1900
```

Comment:
GPTK comment format is `#` not `//`, also what is different about the x55 and 353p vs any other device that they need a different controller mapping?

Reply:
Physical triggers.  Swapping the R1 bumper for the R2 trigger for fire, for example since the layout has the triggers positioned in the "natural" place on the controller.

Reply:
Ahhh okay. Awesome. 😁